### ISSUE TITLE
How-to TPT data workshop

### ISSUE THREAD

**Issue by seltmann** [globalbioticinteractions/globalbioticinteractions#794]
Extend and improve the [Species Interaction Data Workshop] (https://www.globalbioticinteractions.org/interaction-data-workshop) and [How-to page](https://www.globalbioticinteractions.org/how-to) via organizing a TPT workshop. The workshop would be:

1. Be open to anyone
2. Facilitated by @EMTuckerLabUMMZ, @jhpoelen, and @seltmann 
3. Focus on data reviewing and improve existing methods to access TPT data
4. Document workshop results

Workshop topics could include:
1. Creating and reviewing GloBI data index configurations for your data
2. Accessing and reviewing your data
3. Accessing all GloBI data based on your interests
4. Name alignment
5. Providing feedback and community participation

Possible dates? Summer 2022?

**Comment by EMTuckerLabUMMZ** [comment]
Sounds great! June is a bit hectic for me, but July onward would be good I
think.


----------
Erika Tucker
MPM | BON
Bug.News

**Comment by jhpoelen** [comment]
< 20 July would work for me. After that I'll likely be traveling / hosting.

**Comment by jhpoelen** [comment]
@EMTuckerLab would it be an idea to announce a workshop during this Friday's report out. Then, we can say: if you'd like to learn more about the indexed data, name alignment or . . .  come on over and share your interests and skills at the TPT data workshop.

**Comment by jhpoelen** [comment]
Would this TPT data workshop be similar to:

> 22 June 2022 - Erika Tucker of TPT led a workshop on interpreting biotic associations as part of the Entomology Collections Workshop (ECM) 2022. Tucker, Erika, Poelen, Jorrit, & Seltmann, Katja. (2022, June 23). globalbioticinteractions/ecm-workshop:. Zenodo. https://doi.org/10.5281/zenodo.6704580. Also see https://globalbioticinteractions.org/ecm-workshop.

or did you have anything else in mind?

**Comment by EMTuckerLab** [comment]
@Jorrit Poelen ***@***.***> Do you mean a new workshop, or adding
to the online lesson plan? I plan to bring up the how-to page as well as
the workshop lesson plan we put together, but don't go into a lot of detail
(6-minutes goes by fast!). I know we had talked about another workshop at
some point, but didn't a date had been set?

----------
Erika Tucker, PhD (s*he/her/hers*)
Milwaukee Public Museum | TPT  <https://parasitetracker.org/>Taxonomy
<https://parasitetracker.org/>Digitization Manager
<https://parasitetracker.org/>
Biodiversity Outreach Network (BON) <https://www.bon-earth.org/> | Research
Associate <https://www.bon-earth.org/about-us> | S2 Administrative Manager
<https://www.bon-earth.org/_files/ugd/68d22d_372c2df8960041a1a5328b5e43c7b3c3.pdf>
*Entomologist, Museum Specialist, & Researcher*
ORCiD 0000-0002-8822-2315 <https://orcid.org/0000-0002-8822-2315>

Check it out!
   *Bug.News <http://bug.news/> *(blog)
   TPT Resource Hub <https://bit.ly/TPTresourceHub> (taxonomy)
   GloBI How-To <https://www.globalbioticinteractions.org/how-to> (help
page)
   Entomological Resources <https://www.bon-earth.org/products-resources>
(supplies, DIY, etc.)
   BugFlow <https://entcollnet.github.io/BugFlow/> (digitization workflows)
   *Brood X & Spotted Lanternfly Jewelry*
<https://www.earthengypsydesigns.com/category/bugs>!

<https://www.earthengypsydesigns.com/product/17yr-cicada-wings-fw-necklace-stones>
<https://www.earthengypsydesigns.com/product/17yr-cicada-wings-h2>
<https://www.earthengypsydesigns.com/product/spotted-lanternfly-framed-teardrop-hw>
<https://www.earthengypsydesigns.com/product/spotted-lanternfly-earrings-fw-pearl>

**Comment by EMTuckerLab** [comment]
Yes - same here. I don't know that we actually scheduled it - just
discussed it...


----------
Erika Tucker
MPM | BON
Bug.News
